.github/workflows/codeql.yml

```
@@ -0,0 +1,57 @@
+    timeout-minutes: 360
+    
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+        - language: java-kotlin
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 8

(indentation)